jobs/structural/structural_main.sh.mustache

```diff
@@ -93,12 +93,16 @@ rotate () {
   mv -v "$1" "$1.${suffix}"
 }
 
+module load python/2.7
+source /users/PZS0645/wiag/local-owens/emc2/warp3d/17.8.2/warp3d2exii/venv/bin/activate
```

Comment:
WE have `setenv          WARP3D_VENV     $root/warp3d2exii/venv` in the module file, shouldn't it load the module then source `$WARP3D_VENV/bin/activate`?

Reply:
I guess `prereq          python/2.7`. I did not know you could execute `source /users/PZS0645/wiag/local-owens/emc2/warp3d/17.8.2/warp3d2exii/venv/bin/activate` in the module file (or perhaps `source $WARP3D_HOME/warp3d2exii/venv/bin/activate`) but if you can do that then sure.

Reply:
I think you can do stuff like that, but I'd just meant the warp3d module sets the ENV variable `WARP3D_VENV` in the module.

I only suggest it to avoid having the full path there (having `17.8.2`) in the source command because almost assuredly we'll forget next time we have to update.